# sys-libs/musl: undefined reference to __stack_chk_fail_local (x86).
# https://www.openwall.com/lists/musl/2018/09/11/2

diff --git a/configure.ac b/configure.ac
index 8a7cee55b8..67d5b562c9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -831,21 +831,21 @@ AS_IF([test "$GCC" = yes], [
     RUBY_TRY_CFLAGS(-D_FORTIFY_SOURCE=2, [RUBY_APPEND_OPTION(XCFLAGS, -D_FORTIFY_SOURCE=2)])

     # -fstack-protector
-    AS_CASE(["$target_os"],
-    [mingw*], [
-	stack_protector=no
-    ])
-    AS_IF([test -z "${stack_protector+set}"], [
-	RUBY_TRY_CFLAGS(-fstack-protector, [stack_protector=yes], [stack_protector=no])
-	AS_IF([test "x$stack_protector" = xyes], [
-	    RUBY_TRY_LDFLAGS(-fstack-protector, [], [stack_protector=broken])
-	])
-    ])
-    AS_IF([test "x$stack_protector" = xyes], [
-	RUBY_APPEND_OPTION(XCFLAGS, -fstack-protector)
-	RUBY_APPEND_OPTION(XLDFLAGS, -fstack-protector)
-	RUBY_APPEND_OPTION(LDFLAGS, -fstack-protector)
-    ])
+    #AS_CASE(["$target_os"],
+    #[mingw*], [
+    #	stack_protector=no
+    #])
+    #AS_IF([test -z "${stack_protector+set}"], [
+    #	RUBY_TRY_CFLAGS(-fstack-protector, [stack_protector=yes], [stack_protector=no])
+    #	AS_IF([test "x$stack_protector" = xyes], [
+    #	    RUBY_TRY_LDFLAGS(-fstack-protector, [], [stack_protector=broken])
+    #	])
+    #])
+    #AS_IF([test "x$stack_protector" = xyes], [
+    #	RUBY_APPEND_OPTION(XCFLAGS, -fstack-protector)
+    #	RUBY_APPEND_OPTION(XLDFLAGS, -fstack-protector)
+    #	RUBY_APPEND_OPTION(LDFLAGS, -fstack-protector)
+    #])

     AS_CASE("${compress_debug_sections:-zlib}",
     [none|no], [], [
